target/linux/ath79/image/nand.mk

```diff
@@ -305,20 +305,40 @@ define Device/netgear_wndr4500-v3
 endef
 TARGET_DEVICES += netgear_wndr4500-v3
 
-define Device/zte_mf286
+define Device/zte_mf286_common
   SOC := qca9563
   DEVICE_VENDOR := ZTE
-  DEVICE_MODEL := MF286
-  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ledtrig-usbport kmod-ath10k-ct \
```

Comment:
You could leave the "common/shared" `DEVICE_PACKAGES` as is in here.

And then add device specific packages via  `DEVICE_PACKAGES += device-specific-package` in the variants that need it.

Reply:
Done.